vscode-dotnet-runtime-library/distro-data/distro-support.json

```diff
@@ -87,5 +87,61 @@
 				]
 			}
 		]
+	},
+	"Red Hat Enterprise Linux": {
+		"installCommand": "sudo dnf install {0} -y",
+		"uninstallCommand": "sudo dnf remove {0} -y",
+		"updateCommand": "sudo dnf update {0} -y",
+		"isInstalledCommand": "sudo dnf list --installed {0}",
+		"packageLookupCommand": "yum list installed {0}",
+		"expectedDistroFeedInstallDirectory" : "/usr/lib64/dotnet/sdk",
```

Comment:
Copying from https://github.com/nagilson/vscode-dotnet-runtime/pull/7 : Will it go into lib64 on a 32-bit or arm64 version of redhat? or does this folder name change based on the cpu architecture?